Octave provides a replacement for the XERBLA function. The dynamic OpenBLAS
library should use that implementation of XERBLA (instead of its own).
Weak symbols in dynamic libraries cannot be used on Windows for that use-case.
Remove the implementation of the XERBLA function from the OpenBLAS library as
a workaround.

diff -ur OpenBLAS-0.3.34.orig/driver/others/Makefile OpenBLAS-0.3.34/driver/others/Makefile
--- OpenBLAS-0.3.34.orig/driver/others/Makefile	2026-07-17 09:27:06.941068684 -0400
+++ OpenBLAS-0.3.34/driver/others/Makefile	2026-07-17 09:27:40.472978882 -0400
@@ -1,7 +1,7 @@
 TOPDIR	= ../..
 include ../../Makefile.system
 
-COMMONOBJS	 = memory.$(SUFFIX) xerbla.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX) blas_server_callback.$(SUFFIX)
+COMMONOBJS	 = memory.$(SUFFIX) c_abs.$(SUFFIX) z_abs.$(SUFFIX) openblas_set_num_threads.$(SUFFIX) openblas_get_num_threads.$(SUFFIX) openblas_get_num_procs.$(SUFFIX) openblas_get_config.$(SUFFIX) openblas_get_parallel.$(SUFFIX) openblas_error_handle.$(SUFFIX) openblas_env.$(SUFFIX) blas_server_callback.$(SUFFIX)
 
 #COMMONOBJS	+= slamch.$(SUFFIX) slamc3.$(SUFFIX) dlamch.$(SUFFIX)  dlamc3.$(SUFFIX)
 
@@ -96,24 +96,24 @@
 
 ifeq ($(DYNAMIC_ARCH), 1)
 ifeq ($(ARCH),arm64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_arm64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_arm64.$(SUFFIX)
 else
 ifeq ($(ARCH),power)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_power.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_power.$(SUFFIX)
 else
 ifeq ($(ARCH),zarch)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_zarch.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_zarch.$(SUFFIX)
 else
 ifeq ($(ARCH),mips64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_mips64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_mips64.$(SUFFIX)
 else
 ifeq ($(ARCH),loongarch64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_loongarch64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_loongarch64.$(SUFFIX)
 else
 ifeq ($(ARCH),riscv64)
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic_riscv64.$(SUFFIX) detect_riscv64.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic_riscv64.$(SUFFIX) detect_riscv64.$(SUFFIX)
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) dynamic.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) dynamic.$(SUFFIX)
 endif
 endif
 endif
@@ -121,12 +121,9 @@
 endif
 endif
 else
-HPLOBJS = memory.$(SUFFIX) xerbla.$(SUFFIX) parameter.$(SUFFIX)
+HPLOBJS = memory.$(SUFFIX) parameter.$(SUFFIX)
 endif
 
-xerbla.$(SUFFIX) : xerbla.c
-	$(CC) $(CFLAGS) -c $< -o $(@F)
-
 dynamic%$(SUFFIX) : dynamic%c
 	$(CC) $(CFLAGS) -c $< -o $(@F)
 
@@ -220,9 +217,6 @@
 detect_riscv64.$(SUFFIX): detect_riscv64.c
 	$(CC) $(CFLAGS) -c -march=rv64imafdcv $< -o $(@F)
 
-xerbla.$(PSUFFIX) : xerbla.c
-	$(CC) $(PFLAGS) -c $< -o $(@F)
-
 parameter.$(PSUFFIX) : parameter.c ../../param.h
 	$(CC) $(PFLAGS) -c $< -o $(@F)
 
diff -ur OpenBLAS-0.3.34.orig/exports/gensymbol OpenBLAS-0.3.34/exports/gensymbol
--- OpenBLAS-0.3.34.orig/exports/gensymbol	2026-07-17 09:27:06.942647930 -0400
+++ OpenBLAS-0.3.34/exports/gensymbol	2026-07-17 09:40:20.142159509 -0400
@@ -4117,8 +4117,6 @@
                 "$symbolprefix" "$obj" "$symbolsuffix" "$obj" "$count"
             count=`expr $count + 1`
         done
-        printf '\t%sopenblas_set_xerbla%s=openblas_set_xerbla  @%s\n' \
-            "$symbolprefix" "$symbolsuffix" "$count"
     ;;
 
     win2khpl)
diff -ur OpenBLAS-0.3.34.orig/exports/gensymbol.pl OpenBLAS-0.3.34/exports/gensymbol.pl
--- OpenBLAS-0.3.34.orig/exports/gensymbol.pl	2026-07-17 09:27:06.942647930 -0400
+++ OpenBLAS-0.3.34/exports/gensymbol.pl	2026-07-17 09:39:17.305490248 -0400
@@ -4058,7 +4058,7 @@
         print "\t",$symbolprefix,$objs,$symbolsuffix,"=$objs","  \@", $count, "\n";
         $count ++;
     }
-    print "\t",$symbolprefix,"openblas_set_xerbla",$symbolsuffix,"=openblas_set_xerbla  \@",$count,"\n";
+    #print "\t",$symbolprefix,"openblas_set_xerbla",$symbolsuffix,"=openblas_set_xerbla  \@",$count,"\n";
 
     exit(0);
 }
